src/Votes.jsx

```diff
@@ -2,44 +2,45 @@ import React, { useEffect, useState } from "react";
 import axios from "axios";
 
 const Vote = ({ article }) => {
-    const currVotes = article.votes
-  const [votes, setVotes] = useState(currVotes);
+  const [votes, setVotes] = useState(article.votes);
```

Comment:
I think it might be nicer to have state here that represents the amount of new votes from a user to allow a bit more control on what they can and can't do. For example, I can't undo a vote in either direction, but I think this is more a point of refactor ad this is ok for now :)